tools/kleaver/main.cpp

```diff
@@ -382,11 +382,9 @@ static bool printInputAsSMTLIBv2(const char *Filename,
 }
 
 int main(int argc, char **argv) {
-#if LLVM_VERSION_CODE >= LLVM_VERSION(13, 0)
-  KCommandLine::HideOptions(llvm::cl::getGeneralCategory());
-#else
-  KCommandLine::HideOptions(llvm::cl::GeneralCategory);
-#endif
+  std::set<llvm::cl::OptionCategory *> kleaverCategories = {&ExprCat,
+                                                            &SolvingCat};
+  KCommandLine::KeepOnlyCategories(kleaverCategories);
```

Comment:
Here, especially, we can just pass the initializer list directly to the function
```suggestion
  KCommandLine::KeepOnlyCategories({&ExprCat, &SolvingCat});
```